src/components/Error404.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+import Planet404 from "../assets/vectors/404-planet.svg"
+
+import GalaxyBg from "../assets/images/GalaxyBg.png";
+
+// eslint-disable-next-line react/display-name
+export default function() {
+    return (
+        // eslint-disable-next-line react/react-in-jsx-scope
+        // tslint:disable-next-line:max-line-length
+        <div style={{width: "100%", height: "100%", display: "flex", justifyContent: "center", alignItems: "center", flexDirection: "column", backgroundImage: `url(${GalaxyBg})`}}>
```

Comment:
I think that styles should be applied in different way. @DiD3n What should be used there?

Reply:
@DiD3n Can you provide an example code or documentation on what should be used instead?

Reply:
here u go https://www.javascriptstuff.com/css-modules-by-example/